detox/src/devices/Fbsimctl.js

```diff
@@ -173,6 +173,14 @@ class Fbsimctl {
     await this._execFbsimctlCommand(options);
   }
 
+  async resetContentAndSettings(udid) {
```

Comment:
Not sure it fits `launchApp` since it has no relation to the app, it's more of a device function (like `setLocation` or `setOrientation`)